mne/io/cnt/cnt.py

```diff
@@ -60,6 +61,12 @@ def read_raw_cnt(input_fname, montage, eog=(), misc=(), ecg=(), emg=(),
         Names of channels or list of indices that should be designated
         EMG channels. If 'auto', the channel names containing 'EMG' are used.
         Defaults to empty tuple.
+    n_bytes : int
+        Defines the number of bytes the data is read in (should be 2 for int16
+        and 4 for int32). Defaults to 2.
```

Comment:
You might be able to do it in Octave. But in any case, what needs to be done? Read in some data to EEGLAB, write out CNT if possible?